rasa/core/policies/memoization.py

```diff
@@ -45,44 +65,42 @@ class MemoizationPolicy(Policy):
     training stories for this, use AugmentedMemoizationPolicy.
     """
 
-    ENABLE_FEATURE_STRING_COMPRESSION = True
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        """Returns the default config (see parent class for full docstring)."""
+        # please make sure to update the docs when changing a default parameter
+        return {
+            "enable_feature_string_compression": True,
+            "use_nlu_confidence_as_score": False,
+            POLICY_PRIORITY: MEMOIZATION_POLICY_PRIORITY,
+            POLICY_MAX_HISTORY: DEFAULT_MAX_HISTORY,
+        }
 
-    USE_NLU_CONFIDENCE_AS_SCORE = False
+    required_packages = []
```

Comment:
you actually don't need that need one (it's implemented as staticmethod as well but it has a default)
```suggestion
```